modules/nw-modifying-operator-install-config.adoc

```diff
@@ -50,20 +45,34 @@ endif::ignition-config[]
 +
 [source,terminal]
 ----
-$ ./openshift-install create manifests --dir=<installation_directory> <1>
+$ ./openshift-install create manifests --dir=<installation_directory>
 ----
-<1> For `<installation_directory>`, specify the name of the directory that
-contains the `install-config.yaml` file for your cluster.
++
+--
+where:
+
+`<installation_directory>`:: Specifies the name of the directory that contains the `install-config.yaml` file for your cluster.
+--
 
-. Create a file that is named `cluster-network-03-config.yml` in the
-`<installation_directory>/manifests/` directory:
+. Create a stub manifest file for the advanced network configuration that is named `cluster-network-03-config.yml` in the `<installation_directory>/manifests/` directory:
 +
 [source,terminal]
 ----
-$ touch <installation_directory>/manifests/cluster-network-03-config.yml <1>
+$ cat <<EOF > <installation_directory>/manifests/cluster-network-03-config.yml
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+EOF
 ----
-<1> For `<installation_directory>`, specify the directory name that contains the
++
+--
+where:
+
+`<installation_directory>`:: Specifies the directory name that contains the
 `manifests/` directory for your cluster.
+--
 +
 After creating the file, several network configuration files are in the
 `manifests/` directory, as shown:
```

Comment:
(Not sure there's any reason to point this out?)

Reply:
@danwinship, we try to include verification steps, but a `touch` is such a basic operation, you're right that this is probably too verbose. I doubt anyone has ever actually done this step. I'll omit it.